nixos/tests/acme.nix

```diff
@@ -60,7 +60,7 @@ in import ./make-test-python.nix ({ lib, ... }: {
       networking.firewall.allowedTCPPorts = [ 80 443 ];
 
       # OpenSSL will be used for more thorough certificate validation
-      environment.systemPackages = [ pkgs.openssl ];
+      environment.systemPackages = [ pkgs.openssl pkgs.coreutils ];
```

Comment:
`coreutils` is already in systemPackages by default, duplicating it here should (hopefully!?) not be necessary

Reply:
Huh! Yeah that worked. Will remove.